cypress/integration/dataSpec.js

```diff
@@ -2,6 +2,6 @@ import { testNonHTMLResponseCode } from '../support/testHelper';
 
 describe('Static Articles data', () => {
   it('should return a 200 status code', () => {
-    testNonHTMLResponseCode('/data/c0000000001o.json', 200);
+    testNonHTMLResponseCode('/data/news/c0000000027o.json', 200);
```

Comment:
This has to be changed because Simorgh now returns a 404 for incorrect requests, the test would have previously always passed as SImorgh always returned a 200.